Sources/Extensions/UIKit/UIViewControllerExtensions.swift

```diff
@@ -77,7 +77,27 @@ public extension UIViewController {
         present(alertController, animated: true, completion: nil)
         return alertController
     }
-
+    
+    /// SwifterSwift: Helper method to add a UIViewController as a childViewController.
+    ///
+    /// - Parameters:
+    ///   - child: the view controller to add as a child
+    public func add(_ child: UIViewController) {
+        addChildViewController(child)
+        view.addSubview(child.view)
+        child.didMove(toParentViewController: self)
+    }
+    
+    /// SwifterSwift: Helper method to remove a UIViewController from its parent.
```

Comment:
Here too, remove do not make clear that is from the parent and perform this operation, I would suggest removeFromParentViewController, but it already the method of UIKit. Maybe fullyRemoveFromParentViewController().

Reply:
This is tough, I really don't like the name `fullyRemoveFromParentViewController`

Reply:
That's true, it's not so good 😆 But this one is tough, all the naming to make it more clear I could think is too big 😭 
What about `removeFromParentViewControllerWithView` or `removeViewAndControllerFromParentViewController`?